prototypes/examples/create-react-app/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "create-react-app",
```

nit: would be more consistent
```suggestion
  "name": "web-bundles-prototype-app",
```